api/v1/openapi.yaml

```diff
@@ -960,6 +975,13 @@ definitions:
         "$ref": "#/definitions/Labels"
       to:
         "$ref": "#/definitions/Labels"
+      dports:
+        description: |
+          List of Layer 4 port and protocol pairs which the identity will use for
```

Comment:
This description is unclear. What identity? Let's be very precise:
List of Layer 4 port and protocol pairs which will be used in communication from the source identity to the destination identity.